### DISCUSSION TITLE
Support for CB3S Tuya chips?

### DISCUSSION THREAD

**Discussion by serial8killer** [RavenSystem/esp-homekit-devices#2207] [General]
Hi, I bought a couple of led strip controllers that I thought that were ESP chip, but they ended up using the CB3S Tuya chip.
https://developer.tuya.com/en/docs/iot/cb3s?id=Kai94mec0s076
Is this chip supported by HAA?

**Comment by PhaseAngle** [comment]
There are other projects that support them but the only one I found required access to the uart and was pretty much a Tasmota clone.
Since most of these devices its to hard to flash them in place I have always swapped the boards out
